packages/dashboard/e2e/README.md

```diff
@@ -0,0 +1,36 @@
+# E2E workflow
+
+This is to document the flow and interaction of the e2e services in the github environment when:
+
+- starting up the services
+- running the tests
+
+## Starting up the services
+
+Below is a diagram representing the flow of commands when running `npm run test:e2e`
+
+![Flow of commands diagram](documentation-utils/e2e-E2e-start-process.png)
+
+## Container and network interactions
+
+Below is a diagram of how the containers and networks interact when up and running
+
+![Containers and network interaction diagram](documentation-utils/e2e-Container-networks.png)
```

Comment:
Can we use a different background for the container boxes? Don't know if its my screen but the dashboard container particularly is very hard to read.